torch/distributed/pipeline/sync/microbatch.py

```diff
@@ -173,7 +190,11 @@ def scatter(*inputs, chunks: int) -> List[Batch]:
         else:
             # Replicate non-tensors.
             for i in range(chunks):
-                batches[i].append(input)
+                if isinstance(input, NoChunk):
```

Comment:
nit: update the comment on 191 to say replicate non-tensors, or tensors that have been marked with `NoChunk`?